eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/RestExecutedView.php

```diff
@@ -133,5 +128,39 @@ public function visit( Visitor $visitor, Generator $generator, $data )
 
         $generator->endObjectElement( 'View' );
     }
-}
 
+    private function mapSearchHit( SearchHit $searchHit )
+    {
+        if ( $searchHit->valueObject instanceof Content )
+        {
+            return $this->mapContentSearchHit( $searchHit->valueObject );
+        }
+
+        if ( $searchHit->valueObject instanceof Location )
+        {
+            return $this->mapLocationSearchHit( $searchHit->valueObject );
+        }
+    }
+
+    /**
+     * @return RestContent
+     */
+    private function mapContentSearchHit( Content $content )
+    {
+        return new RestContentValue(
+            $content->contentInfo,
```

Comment:
without knowing the details, isn't this redundant? i mean, in the third parameter we're passing ```$content``` and in the first ```$content->contentInfo```. Couldn't we have only one? (i'd say ```$content```)